glide-core/redis-rs/redis/src/aio/mod.rs

```diff
@@ -84,6 +87,12 @@ pub trait ConnectionLike {
 
     /// Returns the state of the connection
     fn is_closed(&self) -> bool;
+
+    /// Get the connection availibility zone
+    fn get_az(&self) -> Option<String>;
+
+    /// Set the connection availibility zone
+    fn set_az(&mut self, az: Option<String>);
```

Comment:
default impl will make it possible to omit all other default impls
```suggestion
    fn set_az(&mut self, az: Option<String>) {}
```